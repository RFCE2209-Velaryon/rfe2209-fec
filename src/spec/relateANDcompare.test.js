import renderer from 'react-test-renderer';
import React from 'react';

import Related from '../components/relateANDcompare/relateANDcompare.jsx';
import RelatedCard from '../components/relateANDcompare/relatedCard.jsx';
import CompareModal from '../components/relateANDcompare/compareModal.jsx';
import Outfit from '../components/relateANDcompare/yourOutfit.jsx';

describe('Components Exist', () => {
  it('renders Related', () => {
    const tree = renderer
      .create(<Related />)
      .toJSON();
      console.log(tree)
      expect(tree).toMatchSnapshot();
   });
   it('renders RelatedCard', () => {
    const tree = renderer
      .create(<RelatedCard />)
      .toJSON();
      console.log(tree)
      expect(tree).toMatchSnapshot();
   });
   it('renders CompareModal', () => {
    const tree = renderer
      .create(<CompareModal />)
      .toJSON();
      console.log(tree)
      expect(tree).toMatchSnapshot();
   });
   it('renders Outfit', () => {
    const tree = renderer
      .create(<Outfit />)
      .toJSON();
      console.log(tree)
      expect(tree).toMatchSnapshot();
   });
});