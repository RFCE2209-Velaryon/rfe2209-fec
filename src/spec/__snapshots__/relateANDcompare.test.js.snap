// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h1>
  Related Items Component
</h1>
`;
